src/table/__integ__/resizable-columns.test.ts

```diff
@@ -19,7 +20,6 @@ const wrapper = createWrapper();
 const tableWrapper = wrapper.findTable();
 // All the columns fit in the viewport, which make it easier to test the columns' widths
 const defaultScreen = { width: 1680, height: 800 };
-const prepareCoordinate = (num: number) => Math.round(num);
```

Comment:
I am also not quite following why you disliked this function and replaced with inline.

For the context, rounding is needed, because WebDriver API expects integer numbers only and fails when a floating value is provided.

Reply:
ok, without that contextual comment it is not very useful, I agree